compiler/circle-mpqsolver/src/utils/ErrorApproximator.cpp

```diff
@@ -0,0 +1,334 @@
+/*
```

Comment:
Q) do we need to put these files in `utils` folder? why not just put them in `src` folder?

Reply:
> Q) do we need to put these files in `utils` folder? why not just put them in `src` folder?

Ok. Seems like 'methods' folder should also be removed.